realm/realm-library/src/syncIntegrationTest/java/io/realm/objectserver/utils/HttpUtils.java

```diff
@@ -60,20 +68,32 @@ public static void startSyncServer() throws Exception {
     // Checking the server
     private static boolean waitAuthServerReady() throws InterruptedException {
         int retryTimes = 50;
+
+        // Dummy invalid request, which will trigger a 400 (BAD REQUEST), but indicate the auth
+        // server is responsive
         Request request = new Request.Builder()
-                .url(Constants.AUTH_SERVER_URL)
+                .post(RequestBody.create(MediaType.parse("application/json; charset=utf-8"), ""))
+                .url(Constants.AUTH_URL)
```

Comment:
Why change this? Also it is strange that i can see from CI log the ROS docker has been started/stopped correctly:
```
+ docker logs ros
info: Integration test server listening on: 127.0.0.1:8888
info: Starting sync server in  /tmp/ros117014-6-g5ntk9
info: 
info: stdout: 2017-01-14T07:37:58.674Z - [32minfo[39m: Logging to file /tmp/realm-sync.log at level 'all'.

info: Sync server stopped
info: child process exited with code 0
info: Sync server stopped
```

But the CI still failed because of `waitAuthServerReady` return false.